Failed to bundle `npm` as a dependency: Cannot find module 'node-gyp/bin/node-gyp'

I can't seem to bundle `npm` as a runtime dependency.

This is necessary to get [Package Phobia on Now 2.0](https://github.com/styfle/packagephobia/pull/217).

## Steps to reproduce

```sh
git clone https://github.com/styfle/ncc-bug-npm
cd ncc-bug-npm
ncc build index.js
node dist/index.js
```

## Error output

```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'node-gyp/bin/node-gyp'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at resolveFileName (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:95521:39)
    at resolveFrom (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:95535:9)
    at module.exports.module.exports (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:95538:41)
    at Object.<anonymous> (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:33356:31)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
    at Object.<anonymous> (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:47941:19)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
    at Object.<anonymous> (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:73093:32)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
    at Object.module.exports.module.exports (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:92854:14)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
    at Object.<anonymous> (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22110:19)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
    at Object.<anonymous> (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:57109:13)
    at __webpack_require__ (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:22:30)
```

I also have a branch called [npm-commands](https://github.com/styfle/ncc-bug-npm/tree/npm-commands) in the same repo that gets a different error:

```
/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:28613
	throw e;
	^

Error: Cannot find module '/Users/styfle/Desktop/ncc-bug-npm/dist/lib/install.js'
    at webpackEmptyContext (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:28611:10)
    at Object.get [as install] (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:33054:41)
    at EventEmitter.npm.load (/Users/styfle/Desktop/ncc-bug-npm/dist/index.js:40778:26)
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "@google-cloud/bigquery": "^2.0.1",
     "@google-cloud/firestore": "^0.19.0",
     "@sentry/node": "^4.3.0",
-    "@zeit/webpack-asset-relocator-loader": "0.1.8",
+    "@zeit/webpack-asset-relocator-loader": "0.1.10",
     "analytics-node": "^3.3.0",
     "apollo-server-express": "^2.2.2",
     "arg": "^2.0.0",

